app/src/app/[lng]/layout.tsx

```diff
@@ -24,7 +24,7 @@ export default function RootLayout(props: {
   const { lng } = use(props.params);
 
   return (
-    <html lang={lng} dir={dir(lng)}>
+    <html lang={lng} dir={dir(lng)} suppressHydrationWarning>
```

Comment:
Let's fix the hydration issue by making sure the server renders the same markup as the client or we use a suspense (but ideally not at the root of the page as it would prevent full server rendering)